src/dev/build/tasks/nodejs/node_download_info.ts

```diff
@@ -15,7 +15,12 @@ export function getNodeDownloadInfo(config: Config, platform: Platform) {
   const arch = platform.getNodeArch();
   let variants = ['default'];
   if (platform.isLinux()) {
-    variants = ['glibc-217'];
+    // CI_USE_POINTER_COMPRESSION is an override for running all tests with pointer compression enabled
+    if (Boolean(process.env.CI_USE_POINTER_COMPRESSION) && !platform.isServerless()) {
```

Comment:
The complexity here is to maintain both node.js builds in the serverless distribution, otherwise our jest tests lose their determinism.